py/desispec/scripts/procexp.py

```diff
@@ -1,21 +1,15 @@
-
 """
-This script processes an exposure by applying fiberflat, sky subtraction,
-spectro-photometric calibration depending on input.
+This script computes the template SNR
```

Comment:
Something has gone wrong here.  This script originally was about processing the exposure to apply all of the calibration steps and write the final cframe files.  The comment here implies that it now only computes the template SNR, and many of the other imports have been removed, e.g. `from desispec.fluxcalibration import apply_flux_calibration`, but downstream in this code `apply_flux_calibration` is still called.  i.e. this code will not run in the pipeline as-is.

Reply:
Yes, agreed.  Sorry this has crept in by accident.  It should be a feature added to procexp, and was running as such recently.  I'll fix that up.